regress/pesterTests/SSH.Tests.ps1

```diff
@@ -296,6 +296,18 @@ Describe "E2E scenarios for ssh client" -Tags "CI" {
             iex "cmd /c `"ssh -o UserKnownHostsFile=`"$kh`" -o StrictHostKeyChecking=no test_target hostname 2>&1`""
             @(Get-Content $kh).Count | Should Be 1
         }
+
+        It "ProxyCommand with absolute path" {            
+            & cmd /c "ssh -o ProxyCommand=`"cmd.exe /c echo Invalid proxy 1>&2`" abc 2>$stderrFile"
+            $stderrFile | Should Contain "Invalid proxy"
+            $stderrFile | Should Contain "Connection closed by remote host"
+        }
+
+        It "ProxyCommand with file name" {
```

Comment:
What is this test case specifically targeting ?

Reply:
this one only contains file name instead of full absolute path. it is to validate the file is searched by the system